data/battle_scripts_1.s

```diff
@@ -660,6 +660,7 @@ BattleScript_EffectRemoveTerrain:
 	resultmessage
 	waitmessage B_WAIT_TIME_LONG
 	removeterrain
+	trytorevertmimicry
```

Comment:
why not just include the revert code within `VARIOUS_REMOVE_TERRAIN`?

Reply:
Didn't think about it too much. I'll do it right away :eyes: